src/Elastic.Apm.AspNetFullFramework/Elastic.Apm.AspNetFullFramework.csproj

```diff
@@ -1,62 +1,29 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not sure we want to change this to the new format - not pushing back on this one, but isn't this unrelated?

Reply:
It made it much easier for me to resolve the reference issues I was having as the new format means that Visual Studio and NuGet takes care of everything default instead of having to explicitly state everything in the old format. It also leans up the code and there is no disadvantage to this.